scripts/history/migrate-services.js

```diff
@@ -0,0 +1,212 @@
+import fsApi from 'fs';
```

Comment:
This implementation seems very clear to me but have the downside that every migration created will have commitable changes.

I think this may not be part of this PR but it would be great to have the config loaded from a gitignored file `.migration.json`
Only change needed would be to add the type of repository to it so that the right repository class is used